examples/default/ios/InstabugTests/InstabugSessionReplayTests.m

```diff
@@ -67,7 +67,34 @@ - (void)testGetSessionReplayLink {
     [self.bridge getSessionReplayLink:resolve :reject];
     OCMVerify([self.mSessionReplay sessionReplayLink]);
     [self waitForExpectations:@[expectation] timeout:5.0];
-
 }
 
+- (void)testSetSyncCallback {
+      BOOL expectedValue = YES;
+      
+      XCTestExpectation *completionExpectation = [self expectationWithDescription:@"Completion block should be called with the expected value"];
+      
+      __block BOOL actualValue = NO;
+      
+      OCMExpect([self.mSessionReplay setSyncCallbackWithHandler:[OCMArg checkWithBlock:^BOOL(id obj) {
+          
+          void (^completionBlock)(BOOL) = ^(BOOL boolean) {
+            actualValue = boolean;
+            [completionExpectation fulfill];
+            XCTAssertEqual(actualValue, expectedValue);
+        };
+        
+        self.bridge.sessionEvaluationCompletion = completionBlock;
```

Comment:
I don't think this line belongs to the test, it's part of the `setSyncCallback` implementation already.